csharp/RowOriented/ParquetRowWriter.cs

```diff
@@ -61,6 +109,15 @@ private ParquetRowWriter(ParquetFileWriter parquetFileWriter, WriteAction writeA
             _rows = new TTuple[1024];
         }
 
+        private ParquetRowWriter(ParquetFileWriter parquetFileWriter, WriteAction writeAction, LogicalWriteConverterFactory? logicalWriteConverterFactory = null)
+        {
+            _parquetFileWriter = parquetFileWriter;
+            _parquetFileWriter.LogicalWriteConverterFactory = logicalWriteConverterFactory ?? LogicalWriteConverterFactory.Default;
```

Comment:
Nitpick but I think it's probably clearer to only assign to the converter factory if it's non-null, as it should already be set to the default factory:
```suggestion
            if (logicalWriteConverterFactory != null)
            {
                _parquetFileWriter.LogicalWriteConverterFactory = logicalWriteConverterFactory;
            }
```

Reply:
:+1: yep having separate checks makes sense there